pkgs/test/default.nix

```diff
@@ -71,7 +71,13 @@ with pkgs;
 
   dhall = callPackage ./dhall { };
 
-  makeWrapper = callPackage ./make-wrapper {};
+  makeWrapper = callPackage ./make-wrapper { };
+  makeBinaryWrapper = callPackage ./make-binary-wrapper {
+    makeBinaryWrapper = pkgs.makeBinaryWrapper.override {
+      sanitizers = pkgs.lib.optionals (! (pkgs.stdenv.isDarwin && pkgs.stdenv.isAarch64))
+        [ "undefined" "address" ];
+    };
+  };
```

Comment:
Can we add a short comment why we want this?

Reply:
Do you mean why we want sanitizers, or why we want them for tests only?

Reply:
Why they are tests only and why they are needed to be ovewritten here. That would make the code a lot more understandable.

Reply:
You already added a comment. Great.

Reply:
I added a comment